examples/analytics/emr-on-eks/manifests/coredns-autoscaler.yaml

```diff
@@ -0,0 +1,88 @@
+kind: ServiceAccount
```

Comment:
we should consider adding this to the addon itself

Reply:
Lets create a [cpa](https://github.com/kubernetes-sigs/cluster-proportional-autoscaler) add-on.

Reply:
Added a new add-on now